### PR TITLE
Dashboard: Use timezone dashboard setting when exporting to CSV

### LINKED ISSUE #10497: [bug] Export to CSV should use dashboard timezone setting
I am having an issue exporting data. I think the issue is I am trying to export the data in UTC, but it exports it in local time.

excel
grafana

I discussed this with @DanCech on Slack.
Right now it's going to use the browser timezone for the export because it just uses `moment(<timestamp).format(<format)`

### PR DESCRIPTION
**Pull request by tskarhed**
**Which issue(s) this PR fixes**:
Fixes #10497

**Special notes for your reviewer**:
Timezone settings in dashboard only offers Default, local and UTC, so at the moment I don't think it can export any timezone. Alternatively add it as an export option?

### CODE REVIEW

File: public/app/core/utils/file_export.ts [hunk 1]
```diff
@@ -58,14 +58,23 @@ function formatRow(row, addEndRowDelimiter = true) {
   return addEndRowDelimiter ? text + END_ROW : text;
 }
 
-export function convertSeriesListToCsv(seriesList, dateTimeFormat = DEFAULT_DATETIME_FORMAT, excel = false) {
+export function convertSeriesListToCsv(
+  seriesList,
+  dateTimeFormat = DEFAULT_DATETIME_FORMAT,
+  excel = false,
+  timeZone?: string
```
Comment (torkelo) [inline public/app/core/utils/file_export.ts]: we have a TimeZone type that could be used instead

**Review by torkelo — APPROVED** [review]
LGTM, just one small thing with TimeZone type we have that can be used instead of string

**Review by marefr — COMMENTED** [review]
What's up with the yarn.lock changes? Are those needed for this PR?

**Comment by dehrax** [comment]
@marefr Probably changed after I ran `yarn install`.

>The package.json describes the intended versions desired by the original author, while yarn.lock describes the last-known-good configuration for a given application.
[- Someone](https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for)

So I guess?

File: public/app/core/utils/file_export.ts [hunk 2]
```diff
@@ -75,12 +84,22 @@ export function convertSeriesListToCsv(seriesList, dateTimeFormat = DEFAULT_DATE
...
+  dateTimeFormat = DEFAULT_DATETIME_FORMAT,
+  excel = false,
+  timeZone?: TimeZone
+) {
+  const text = convertSeriesListToCsv(seriesList, dateTimeFormat, excel, timeZone);
   saveSaveBlob(text, EXPORT_FILENAME);
 }
 
-export function convertSeriesListToCsvColumns(seriesList, dateTimeFormat = DEFAULT_DATETIME_FORMAT, excel = false) {
+export function convertSeriesListToCsvColumns(
```
Comment (davkal) [inline public/app/core/utils/file_export.ts]: IMO up to 3 args is fine, better just 2, with the second being an options object, with a proper type definition.

**Review by davkal — COMMENTED** [review]
Needs to remove the lock file update. Otherwise looks good. Tried it out and works.

**Review by davkal — APPROVED** [review]
Looks good!

**Pull request merged**